### PR TITLE
Add getting to know Pehuén Rodriguez article

### PR DESCRIPTION
**Pull request by starbelly**
- [x] I acknowledge my contribution to the website does not assert comparative or superlative differences of one product, project, company or individual over another.

# Screenshots

image

### CODE REVIEW

File: priv/posts/marketing/20210331201645_getting-to-know-us-prodriguez.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+## The beginnings

+
+Pehuén’s professional career started when he decided to study Systems Engineering at UTN (Universidad Nacional de Tecnología). He got his first job in a big company thanks to the university internship program; there, he was in charge of the interpretation of local needs and their translation in terms of requirements/development. After a few years, paradoxically, he decided to pause university and focus on his job.
He spent five years in the same company until one of his future partners at Nayra cooperative asked him to join them, which he agreed happily. At the same time, he decided to explore his creative side by studying Visual Arts. Although he was really enjoying this new career, he felt like there was something that still didn’t quite fit, so he dropped it and kept searching.
After a while, he ran into Electronic Arts, a career offered by UNTREF (Universidad Nacional de Tres de Febrero), where he managed to bring his interests together. He fervently believes that creative thinking is a fundamental part of his profession and that all the way he traveled led him to where he is now.
He has even resumed Communications Systems Engineering this past 2020.
```
Comment (paulo-ferraz-oliveira) [inline priv/posts/marketing/20210331201645_getting-to-know-us-prodriguez.md:22]:
```suggestion
Pehuén’s professional career started when he decided to study Systems Engineering at UTN (Universidad Nacional de Tecnología). He got his first job in a big company thanks to the university internship program; there, he was in charge of the interpretation of local needs and their translation in terms of requirements/development. After a few years, paradoxically, he decided to pause university and focus on his job.  He spent five years in the same company until one of his future partners at Nayra cooperative asked him to join them, which he agreed happily. At the same time, he decided to explore his creative side by studying Visual Arts. Although he was really enjoying this new career, he felt like there was something that still didn’t quite fit, so he dropped it and kept searching.  After a while, he ran into Electronic Arts, a career offered by UNTREF (Universidad Nacional de Tres de Febrero), where he managed to bring his interests together. He fervently believes that creative thinking is a fundamental part of his profession and that all the way he traveled led him to where he is now.  He has even resumed Communications Systems Engineering this past 2020.
```

Comment (paulo-ferraz-oliveira) [reply priv/posts/marketing/20210331201645_getting-to-know-us-prodriguez.md:22]: Hm... it's not easy to spot the difference, it seems. It's missing spaces after `.`.

File: priv/posts/marketing/20210331201645_getting-to-know-us-prodriguez.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+## First steps on the BEAM

+One of Pehuén’s first projects at Nayra was a Canadian reservation and ticketing platform, where he was professionalized in Node.js. There, he also had the opportunity to observe some features and tooling being developed in Elixir. And so, the seed of interest was planted.
It grew up with readings and learning projects and, in the early 2020, a friend cooperative proposed to provide training in Phoenix together: learning with friends to help others learn was more than enough incentive. He didn’t have to think twice.
```
Comment (paulo-ferraz-oliveira) [inline priv/posts/marketing/20210331201645_getting-to-know-us-prodriguez.md:25]:
```suggestion
One of Pehuén’s first projects at Nayra was a Canadian reservation and ticketing platform, where he was professionalized in Node.js. There, he also had the opportunity to observe some features and tooling being developed in Elixir. And so, the seed of interest was planted.  It grew up with readings and learning projects and, in the early 2020, a friend cooperative proposed to provide training in Phoenix together: learning with friends to help others learn was more than enough incentive. He didn’t have to think twice.
```

Comment (paulo-ferraz-oliveira) [reply priv/posts/marketing/20210331201645_getting-to-know-us-prodriguez.md]: This one's more visible. It's a single white space after a `.`.

Comment (paulo-ferraz-oliveira) [reply priv/posts/marketing/20210331201645_getting-to-know-us-prodriguez.md]: There's some more, in the commit, of this type, but I'll let you spot them :)
`\.[^ ]`?

**Review by paulo-ferraz-oliveira — APPROVED** [review]

**Comment by starbelly** [comment]
❤️ @paulo-ferraz-oliveira

**Pull request merged**